### PR TITLE
Add new template file for PRs

### PR DESCRIPTION
**Pull request by OrlinVasilev**
Current file is not correct and it's not applied

Here is example how it will look:
https://github.com/OrlinVasilev/release/pull/11

Signed-off-by: OrlinVasilev <ovasilev@vmware.com>

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Thank you for contributing to Harbor!
+
+# Comprehensive Summary of your change
+
+# Issue being fixed
+Fixes #(issue)
+
+Please indicate you've done the following:
+- [ ] Well Written Title and Summary of the PR
+- [ ] Label the PR as needed. "release-note/ignore-for-release, release-note/new-feature, release-note/update, release-note/enhancement, release-note/community, release-note/breaking-change, release-note/docs, release-note/infra, release-note/deprecation"
```
Comment (Vad1mo) [inline .github/pull_request_template.md:10]: can an external person creating a PR even set label?

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+- [ ] Accepted the DCO. Commits without the DCO will delay acceptance.
```
Comment (Vad1mo) [inline .github/pull_request_template.md:11]: Be honest here: PRs without a DCO are not accepted. At least I never saw one.

**Review by Vad1mo — APPROVED** [review]

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+- [ ] Made sure tests are passing and test coverage is added if needed.
+- [ ] Considered the docs impact and opened a new docs issue or PR with docs changes if needed.
+- [ ] Updated the corresponding documentation in `website/docs/main`.
```
Comment (a-mccarthy) [inline .github/pull_request_template.md:14]: can we update this to be a link to the website repo?

**Review by a-mccarthy — CHANGES_REQUESTED** [review]
@OrlinVasilev Thanks for updating this! just a few some comments. Overall i think its really awesome

**Review by a-mccarthy — APPROVED** [review]
lgtm, thanks orrlin!

File: .github/pull_request_template.md (hunk 1 above)
Comment (wy65701436) [reply .github/pull_request_template.md:10]: the label can be added by the reviewers, and that's fair.

File: .github/pull_request_template.md (hunk 2 above)
Comment (wy65701436) [reply .github/pull_request_template.md:11]: we have CI to guarantee it.

**Review by wy65701436 — APPROVED** [review]
lgtm

**Pull request merged**